.github/workflows/ci.yaml

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
This is the wrong file header and should not pass CI. See the boilerplates directory.

Reply:
Wrong copy/paste, doh.